e2e/wdio/platform/fixtures/appData/dynamic-page-layout-contents.ts

```diff
@@ -0,0 +1,3 @@
+const main_button_title = 'Click to open full screen';
```

Comment:
export const main_button_title = 'Click to open full screen';

Reply:
:+1: